static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -152,7 +135,8 @@ define([
             return 'media:' + eventName;
         }).forEach(function(playerEvent) {
             player.on(playerEvent, function(_, mediaEvent) {
-                ga(gaTracker + '.send', 'event', buildGoogleAnalyticsEvent(mediaEvent, events, canonicalUrl));
+                ga(gaTracker + '.send', 'event', gaHelper.buildGoogleAnalyticsEvent(mediaEvent, events, canonicalUrl,
+                    'guardian-videojs', getGoogleAnalyticsEventAction, mediaEvent.mediaId));
```

Comment:
It depends which implementation you are using.  The hosted implementation uses youtube_id our implementation users atom id.

From hosted on production: 

https://ophan.theguardian.com/a.gif?viewId=ivdvgr9tla1cr8wvdnbe&video={"id":"gu-video-youtube-aR5_bhSO_d0","eventType":"video:content:play"}